backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/spinner/BpkSpinner.kt

```diff
@@ -48,14 +48,27 @@ fun BpkSpinner(
   modifier: Modifier = Modifier,
   style: BpkSpinnerStyle = findBestSpinnerStyleFor(LocalContentColor.current, BpkTheme.colors.isLight),
 ) {
-  CircularProgressIndicator(
-    modifier = modifier.requiredSize(size.dp, size.dp),
-    strokeWidth = 2.dp,
+  BpkSpinner(
+    size = size,
+    modifier = modifier,
     color = when (style) {
       BpkSpinnerStyle.TextPrimary -> BpkTheme.colors.textPrimary
       BpkSpinnerStyle.Disabled -> BpkTheme.colors.textDisabled
       BpkSpinnerStyle.OnDarkSurface -> BpkTheme.colors.textOnDark
-    }
+    },
+  )
+}
+
+@Composable
+internal fun BpkSpinner(
```

Comment:
made an internal way to override the spinner colour as it's needed for the button - we don't want to expose this externally